tests/desktop/test_reviews.py

```diff
@@ -7,6 +7,7 @@
 import pytest
 
 from datetime import datetime
+from pytz import timezone
```

Comment:
Nit: We could make the imports PEP8 while we're here. See http://legacy.python.org/dev/peps/pep-0008/#imports